source/includes/_stories.md.erb

```diff
@@ -22,7 +22,14 @@ curl -X GET "https:/<%= config[:api_endpoint] %>/story"
 }
 ```
 ```javascript
+// The 'start_stories' command is used to create a block of stories and start a campaign.
```

Comment:
`The 'start_stories' command is used to start story campaign programmatically (e.g. on button click)`